scripts/custom/code_to_empty.py

```diff
@@ -0,0 +1,10 @@
+from scripts import utils
+
+
+def code_to_empty(raw_input):
+    """Return None when entry is some code: -1 or (sans)
+    """
+    if utils.is_empty(raw_input) or raw_input == "-1" or raw_input == "(sans)":
```

Comment:
Is `(sans)` something standard?

Reply:
Not standard but very common accross a particular database

Reply:
"particular"